mitmproxy/protocol/socks_client.py

```diff
@@ -0,0 +1,82 @@
+import socket
+
+from netlib import socks
```

Comment:
Very minor nitpick: `from netlib.socks import METHOD, VERSION, CMD, ...` should reduce a lot of text noise further down.